src/components/Graph.vue

```diff
@@ -68,6 +75,13 @@
 		var isLeapYear = (new Date(year, 1, 29).getDate() === 29)
 		return isLeapYear ? 366 : 365
 	}
+
+	const getLink = (year, day) => {
+		const date = getDate(year, day);
+		return props.username != "" ? `https://letterboxd.com/${props.username}/films/diary/for/${date.getFullYear()}/${(date.getMonth() + 1).toString().padStart(2, '0')}/${(date.getDate()).toString().padStart(2, '0')}/` : ""
```

Comment:
Currently, this returns an empty link when there's no username. It would be better if we also disable the link or not render a link if the username is blank (ie, in the `<template>`)

Reply:
I don't know how to do this I don't think you can use v-if lol

Reply:
https://vuejs.org/v2/guide/list.html#v-for-with-v-if you cant mix v-for and v-if

Reply:
i did it